### PR TITLE
Support vanilla VCs.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: CredentialCard.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
             <q-item-label
               class="text-left text-subtitle1"
               lines="2">
-              {{credential.name}}
+              {{credential.name || credential.type[1]}}
```
Comment (dlongley) [inline CredentialCard.vue]: I don't think we can do this, this information doesn't make sense to the user and could even be inaccurate. For now, we should just literally say "Credential" or "Untitled Credential". We're going to need to use other meta information, I think, such as the website where the user retrieved the credential, to display more.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by mattcollier** [comment]
This code is currently in use on wallet master, I've assigned it P1 to get this resolved/merged and released.

**Comment by gannan08** [comment]
Once @dlongley's concerns are addressed. We can merge the code down.

File: CredentialCard.vue (hunk 1 above)
Comment (mattcollier) [reply CredentialCard.vue]: Addressed: d93d49952fdddf521f15dfcfeb79d08e2e3cb5d1

**Review by dlongley — APPROVED** [review]
I'm approving based on my previous review having only one concern which has now been addressed. Thanks!

**Pull request merged**